lib/relatedFiles.js

```diff
@@ -23,6 +23,9 @@ function comparePaths(path1, path2) {
    * @returns {object} - Detect root files result object
    */
 function calculatePath(parentFileName, referencePath) {
+  if (path.isAbsolute(referencePath)) {
```

Comment:
Is the usage of `path` or `pathBrowserify` based on the environment maintained here?

Reply:
The environment, as we do in other modules is handled here: https://github.com/postmanlabs/openapi-to-postman/blob/92e091acb39500b519dc39efd295271609d97e07/lib/relatedFiles.js#L155